api/controllers/FlairController.js

```diff
@@ -145,7 +145,7 @@ module.exports = {
       if (events.length) {
         var then = moment(events[0].createdAt);
         then.add(2, 'minutes');
-        if (then.isAfter(now) && false) {
```

Comment:
Ah yes, that's my fault. I disabled it for testing because it was annoying me. Apologies.